src/header/header.scss

```diff
@@ -429,3 +386,20 @@
     }
   }
 }
+// #endregion Header buttons
+
+// #region Header extras
+@mixin iui-header-divider {
+  width: 1px;
+  height: $iui-baseline * 2;
```

Comment:
I was playing with dividers in the integration sprint and realized that `width: 1px` is not reliable on high resolution screens. Try using borders instead https://ishadeed.com/article/flexbox-separator/